apps/web/src/state/teams/helpers.ts

```diff
@@ -49,7 +49,7 @@ export const getTeams = async (): Promise<TeamsById> => {
           args: [BigInt(i + 1)] as const,
         } as const),
     )
-    const client = publicClient({ chainId: ChainId.BSC })
+    const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```